itou/eligibility/enums.py

```diff
@@ -114,7 +114,15 @@ def for_geiq(cls):
             cls.SIAE_CUI,
         }
 
+    @classmethod
+    def certifiable_by_api_particulier(cls):
+        return frozenset(
+            [
+                AdministrativeCriteriaKind.RSA,
+                AdministrativeCriteriaKind.AAH,
+                AdministrativeCriteriaKind.PI,
```

Comment:
nit:
```suggestion
                cls.RSA,
                cls.AAH,
                cls.PI,
```
pour rester cohérent avec les autres `classmethod` ?